cardinal/ecs/persona.go

```diff
@@ -14,7 +14,10 @@ type CreatePersonaTransaction struct {
 }
 
 // CreatePersonaTx is a concrete ECS transaction.
-var CreatePersonaTx = NewTransactionType[CreatePersonaTransaction]("create-persona")
+var CreatePersonaTx = NewTransactionType[CreatePersonaTransaction](
```

Comment:
I recently made a change to NewTransactionType to take in two generic arguments.